templates/vue/src/components/node-modal/AppearanceForm.vue

```diff
@@ -102,9 +102,16 @@ export default {
       addLockedThumbnail: false,
     }
   },
+  watch: {
+    addThumbnail(addThumbnail) {
+      if (!addThumbnail) {
+        this.node.imageURL = ""
+      }
+    },
+  },
   created() {
-    this.addThumbnail = this.node.imageURL.length > 0
-    this.addLockedThumbnail = this.node.lockedImageURL.length > 0
+    ;(this.addThumbnail = this.node.imageURL.length > 0),
+      (this.addLockedThumbnail = this.node.lockedImageURL.length > 0)
```

Comment:
Not sure why you have added all these symbols. Do they serve a function?

Reply:
huh, that's weird, I didn't write those extra bits.. maybe it was linting? I'll look into that.